ratatool-sampling/README.md

```diff
@@ -85,6 +85,16 @@ Leveraging `--fields=<field1,field2,...>` BigSampler can produce a hash based on
  are in the sample. For example, `--fields=user_id --sample=0.5` will always produce the same sample
  of 50% of users. If multiple records contain the same `user_id` they will all be in or out of the
  sample.
+
+### Reproducing within BigQuery
+Currently, BigSampler defaults to Farmhash, which is also used in BigQuery. When sampling with a seed and one or more fields,
```

Comment:
could you link to [Farmhash](https://github.com/google/farmhash)? Also, it looks like the FarmHash repo has been archived. Considering it's used in BigQuery, it shouldn't be much of an issue, but is it possible that we should explore an alternate hashing algo in the future?

Reply:
I think if we do it would be for the distribution work in #699 , or if farmhash stops working. ATM it's still supported through BQ so not super worried

Reply:
added a link earlier